bazel/repository_locations.bzl

```diff
@@ -664,12 +664,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Rules for using foreign build systems in Bazel",
         project_desc = "Rules for using foreign build systems in Bazel",
         project_url = "https://github.com/bazelbuild/rules_foreign_cc",
-        version = "d54c78ab86b40770ee19f0949db9d74a831ab9f0",
-        sha256 = "e7446144277c9578141821fc91c55a61df7ae01bda890902f7286f5fd2f6ae46",
+        version = "da8952e27c46659866b455de58f5eb3a86185507",
+        sha256 = "5d1e34fc0f4ab4af0e595ee4d9a03f4127701d95c1d521296bf892b8aff903a2",
```

Comment:
I know this has passed CI but do we want to bump to the latest commit? https://github.com/bazelbuild/rules_foreign_cc/commit/6c0c2af3d599f4c23117a5e65e811ebab75bb151

Reply:
done